package.json

```diff
@@ -20,14 +20,46 @@
     "@reach/portal": "0.12.1",
     "@reach/tabs": "0.13.0",
     "@reduxjs/toolkit": "1.4.0",
+    "@rematch/core": "2.2.0",
+    "@rematch/loading": "2.1.2",
+    "@rematch/select": "3.1.2",
+    "@testing-library/jest-dom": "5.11.4",
+    "@testing-library/react": "11.0.2",
+    "@types/jest": "25.1.4",
+    "@types/lodash": "4.14.168",
+    "@types/node": "14.6.4",
+    "@types/react": "16.9.49",
+    "@types/react-dom": "16.9.8",
+    "@types/react-redux": "7.1.9",
+    "@types/react-router-dom": "5.1.5",
+    "@types/react-test-renderer": "16.9.3",
+    "@types/rebass": "4.0.7",
+    "@types/rimraf": "3.0.0",
+    "@types/shelljs": "0.8.8",
+    "@types/styled-components": "5.1.3",
+    "@types/testing-library__jest-dom": "5.9.2",
+    "@types/webpack": "4.41.22",
+    "@types/webpack-env": "1.15.2",
     "@uniswap/token-lists": "1.0.0-beta.30",
     "axios": "0.21.3",
     "big.js": "6.1.1",
     "bignumber.js": "9.0.1",
     "chalk": "4.1.0",
+    "connected-react-router": "6.9.2",
+    "cross-env": "7.0.2",
     "dayjs": "1.10.7",
     "decimal.js-light": "2.5.1",
+    "eslint-config-prettier": "6.11.0",
+    "eslint-plugin-import": "2.22.1",
+    "eslint-plugin-prettier": "3.1.4",
+    "eslint-plugin-react-hooks": "4.1.0",
```

Comment:
There are two definitions of this dependency

Reply:
@hetfly Not only `eslint-plugin-react-hooks` but there are many dependencies that were defined in both `devDependencies` and `dependencies`. Should I keep the dependencies in `devDependencies`? or `dependencies`?

Reply:
@Chunn depends on whether the package is used in the app or just during the build / dev process. If you are not sure, delete the definition from `devDependencies`, keep the one in `dependencies`. Thanks!

Reply:
@hetfly If the version in dependencies is lower than devDependencies, I'll upgrade to the latest version

Reply:
@Chunn I wouldn't do it without proper testing if the latest version has a breaking change (if x is higher in x.y.y version number, there probably is a breaking change)

Reply:
In that case, I'd keep the higher version of the two

Reply:
@hetfly I've updated. Please help me check it 
https://github.com/balancednetwork/balanced-network-interface/pull/980/commits/28c6e5845c7e8cc59998034b6af606c472320887
Thank you!